.idea/vcs.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
file should be deleted and .idea/* ignored in gitignore

Reply:
Yup. These files come from some IDEs, we can add more IDE generated as/if they pop up.